spec/models/apartment_spec.rb

```diff
@@ -1,5 +1,16 @@
 require 'rails_helper'
 
 RSpec.describe Apartment, type: :model do
```

Comment:
This test needs some love. To create an apartment you also need to create a user.